NOTICE.md.meta

```diff
@@ -0,0 +1,7 @@
+fileFormatVersion: 2
+guid: 7eac9b4c9a2ab1f40aab89ba9a3b66a6
```

Comment:
We'll probably need unique metas for each package this gets copied to. Otherwise there will be GUID collisions when multiple packages are installed.

Reply:
good thought.... i suppose we should just check the files in to each package root.